src/BalconyBotanica/Core/DomainObjects/Sunlight.cs

```diff
@@ -0,0 +1,10 @@
+namespace BalconyBotanica.Core.DomainObjects
+{
+    public enum Sunlight
```

Comment:
Just a suggestion on naming (and also variables of this type):

```suggestion
    public enum SunlightRequirements
```